### ISSUE TITLE
Endless waves of enemies.

### ISSUE THREAD

**Issue by Dark-Umbrella** [Anuken/Mindustry-Suggestions#3702]
### Describe what you would like changed, and why.

Previously, the game had endless, ever-increasing waves of enemies, even after clearing the location. Now you can survive the waves and grow resources on a huge scale, discovering and exploring everything that is not limited by location. There was no constant pressure on the player, after defending, you can safely disassemble both walls and guns. The game has become quite boring, because for me, as a player, there is simply no point in following to the next location, when here I can safely feed resources for several tens of hours. 
For example, for the second day I have been in the "careers" location and discovering technologies, although earlier I would have been washed away by a wave of increasingly powerful enemies and spurred on to go through the company further.

### Describe the changes you want to propose. Include possible alternatives.

Inadequate options:
Add a "fill core" button to collect all resources instantly, rather than waiting. 
Or. 
Add time acceleration after clearing the location. 
Adequate options:
To return waves with increasing difficulty after stripping. 
Or. 
Add another restriction on the player's stay on the level.

**Comment by TheRealIvyX** [comment]
> If you want more of a challenge, try battling with actual people.

- itcannotbe, literally 8 minutes ago

**Comment by Halt-alt** [comment]
> ### Describe what you would like changed, and why.
> Previously, the game had endless, ever-increasing waves of enemies, even after clearing the location. Now you can survive the waves and grow resources on a huge scale, discovering and exploring everything that is not limited by location. There was no constant pressure on the player, after defending, you can safely disassemble both walls and guns. The game has become quite boring, because for me, as a player, there is simply no point in following to the next location, when here I can safely feed resources for several tens of hours. For example, for the second day I have been in the "careers" location and discovering technologies, although earlier I would have been washed away by a wave of increasingly powerful enemies and spurred on to go through the company further.
> 
> ### Describe the changes you want to propose. Include possible alternatives.
> Inadequate options: Add a "fill core" button to collect all resources instantly, rather than waiting. Or. Add time acceleration after clearing the location. Adequate options: To return waves with increasing difficulty after stripping. Or. Add another restriction on the player's stay on the level.

I agree that this is a problem within the game. Erekir fixes that by removing core loadouts. However, I have no idea how to fix that on serpulo. You can't simulate every sector at once, so you can't just have endless waves everywhere. Anuke may end up reworking the system the way he did on Erekir.

**Comment by Neil2101** [comment]
are you sure you read what to not suggest in README.md?

**Comment by Neil2101** [comment]
![image](https://user-images.githubusercontent.com/66876298/177629770-246e41ce-7287-47e5-8446-2661442487fa.png)
